libs/components/lookup/src/lib/modules/lookup/lookup.component.scss

```diff
@@ -33,8 +78,12 @@
     }
 
     .sky-tokens {
-      margin-top: -5px;
+      margin-top: var(--sky-override-lookup-tokens-margin-top, -6px);
       margin-bottom: -5px;
```

Comment:
there are a few things not tokenized here, margin-bottom and padding-top below. Is it because they're the same across all themes? Is there any explanation for why it is -5px/-6px? If we can't do some form of calculation, a comment would be helpful